scripts/test

```diff
@@ -5,9 +5,6 @@ USAGE="Run linters and tests."
 source ./scripts/_sourced
 check_for_help_flag "$@"
 
-echo "Running frontend app tests..."
-./scripts/yarn run test
-
```

Comment:
I can't add a comment below, but noting here that the project template shifted from using GH superlinter to megalinter ([this was captured in an issue I prematurely closed](https://github.com/azavea/echo-locator/issues/605)), if we have time in the scope of this card we might want to switch to megalinter to better match the project template. If it feels out of scope I can re-open that issue and we can tackle as part of the setup epic.

Reply:
Ah, got it. How about we re-open that issue? The frontend part is fresh and should be straightforward to switch to a new linter. I suspect backend might be a bit more involved- just a guess.